code/game/gamemodes/endgame/supermatter_cascade/blob.dm

```diff
@@ -79,15 +79,18 @@
 
 	Consume(user)
 
-/turf/unsimulated/wall/supermatter/attackby(obj/item/weapon/W as obj, mob/living/user as mob)
```

Comment:
what on earth is this? A wall made of supermatter?

Reply:
Funny. Can't unswer that question.